pkg/ddevapp/dotddev_assets/providers/README.txt

```diff
@@ -23,7 +23,7 @@ Each provider recipe is a file named `<provider>.yaml` and consists of several m
 * `db_pull_command`: A script that determines how ddev should pull a database. It's job is to create a gzipped database dump in /var/www/html/.ddev/.downloads/db.sql.gz.
 * `files_pull_command`: A script that determines how ddev can get user-generated files from upstream. Its job is to copy the files from upstream to  /var/www/html/.ddev/.downloads/files.
 * `db_push_command`: A script that determines how ddev should push a database. It's job is to take a  gzipped database dump from /var/www/html/.ddev/.downloads/db.sql.gz and load it on the hosting provider.
-* `files_pull_command`: A script that determines how ddev push user-generated files to upstream. Its job is to copy the files from the project's user-files directory ($DDEV_FILES_DIR) to the correct place on the upstream provider.
+* `files_pull_command`: A script that determines how ddev push user-generated files to upstream. Its job is to copy the files from the project's user-files directories ($DDEV_FILES_DIRS) to the correct places on the upstream provider.
```

Comment:
The pull tests don't get executed on a forked repo, so haven't been tested at all. We really need to run this manuallly with all the right stuff, or perhaps (sadly) move this to the ddev/ddev repo. I wish there was an easy way to do that. Or else we can do a follow-up PR to fix pull/push problems that are discovered. (and do it in the ddev repo rather than fork so it gets tested). Alternately, it *may* be possible to add all the necessary secrets to your fork, might work, seems like a lot of work.

Reply:
I guess I can just push to ddev/ddev to execute the tests. Will do later after the other fixes...

Reply:
That's reasonable, important for those tests to pass. If tests don't run on a branch, you can just create a dummy PR with the ddev/ddev branch.